python/lsst/sims/catUtils/utils/ObservationMetaDataGenerator.py

```diff
@@ -248,6 +371,157 @@ def getObservationMetaData(self, obsHistID=None, expDate=None, fieldRA=None, fie
                                                                     for column in self.active_columns if column[2] is not None]))
                                           for pointing in results]
 
-
         return obs_output
 
+    @staticmethod
+    def ObservationMetaDataForPointing(OpSimPointingRecord, columnMap,
+                                       OpSimColumns=None,
+                                       boundLength=1.75, boundType='circle'):
+        """
+        Return instance of ObservationMetaData for an OpSim Pointing record
+        from OpSim.
+
+        Parameters
+        ----------
+        OpSimPointingRecord : Dictionary, mandatory
```

Comment:
Sorry about that. This is correct. It *is* a numpy.recarray, which I realized later when wrote the ObservationMetaDataForPointing and the examples. I initially misunderstood the type, so some of the docstrings/ comments are carried over are carried over, and I need to change those.